.meta/transforms/swimlanes.toml

```diff
@@ -7,6 +7,7 @@ function_category = "filter"
 input_types = ["log"]
 output_types = ["log"]
 requirements = {}
+sub_type = "function"
```

Comment:
`swimlanes` is interesting because it's not really a function or a runtime. It's really a macro that expands to multiple function-based pipelines.

Reply:
That's a good point. I think for this exercise the term "component" might be better? I'm really trying to distinguisn between a set of simple components that you connect to form transformation pipelines, and full runtimes.

Reply:
Yeah that sounds reasonable to me.